Riot/AppDelegate.m

```diff
@@ -1352,7 +1500,15 @@ - (void)handleLocalNotificationsForAccount:(MXKAccount*)account
         MXEvent *event;
 
         // Ignore event already notified to the user
-        if ([self displayedLocalNotificationForEvent:eventId andUser:account.mxCredentials.userId type:nil])
+        if (@available(iOS 10, *))
+        {
+            if ([self displayedNotificationRequestForEvent:eventId andUser:account.mxCredentials.userId type:nil])
```

Comment:
We are not obliged to check for displayed notifications here, as `UNNotificationRequest` have a unique identifier (evendId). The content for a `UNNotificationRequest` with the same identifier will be overwritten.

Reply:
https://github.com/vector-im/riot-ios/pull/2207/commits/93ce20e9cc379762acb626bdecfec62d7b2200f5